frontend/src/audios.html

```diff
@@ -38,9 +53,13 @@
         <span class="hidden sm:inline">Search</span>
       </button>
     </div>
-    <div class="my-3 flex items-center">
+    <div x-data="{ open: false }" class="my-3 flex items-center">
```

Comment:
Move the `x-data` to the button itself.

Reply:
And this component does not need any data.

Reply:
`x-data` moved to button as data less component